doc/Extensions/Applications.md

```diff
@@ -1651,6 +1652,39 @@ SNMP extend script to monitor your Redis Server
 extend redis /etc/snmp/redis.py
 ```
 
+# RRDCached
+
+Install/Setup:
+For Install/Setup Local Librenms RRDCached: Please see [RRDCached](RRDCached.md)
+Securing RRCached: Please see [RRDCached Security](RRDCached-Security.md)
+
+Local Polling:
+For local Polling the 'RRDCached' Application (librenms-localhost) the usual is either to:
+
+1: Use unix:socket as described: [RRDCached](RRDCached.md)
+2: or Monitor thru port listen 42217 as described: [RRDCached Security](RRDCached-Security.md)
```

Comment:
Why are you mentioning RRDCached Security doc so much?  Preferred setup for listening on 42217 is covered in the base RRDCached documentation.

Reply:
no reason. just for users that need that option. want me to remove those 4 lines ?
thx.

Reply:
Just remove them.  As i said, RRDCached Security doc is not very useful for improving rrdcached security, especially since it is a more complicate configuration.

Reply:
ok